src/Backend/Modules/Faq/Actions/Edit.php

```diff
@@ -45,7 +45,7 @@ public function execute(): void
 
     private function getData(): void
     {
-        $this->record = (array) BackendFaqModel::get($this->id);
+        $this->record = BackendFaqModel::get($this->id);
```

Comment:
Don't change anything in the actions, only in the model.

Reply:
this (array) conversion is obsolete, as it is enforced by php7 return type hinting

Reply:
ok